tests/entercommand.js

```diff
@@ -87,6 +87,31 @@ describe( 'EnterCommand', () => {
 				'<p>x</p><p>[]</p><p>y</p>',
 				'<p>x</p><p></p><p>[]</p><p>y</p>'
 			);
+
+			describe( 'copyOnEnter', () => {
+				beforeEach( () => {
+					schema.extend( '$text', { allowAttributes: 'foo' } );
```

Comment:
When I look back and my code, I start to see some issues :D.
Here probably `bar` attribute should be allowed, but defined without `copyOnEnter` property.